pandas/tests/resample/test_datetime_index.py

```diff
@@ -535,11 +535,13 @@ def test_upsample_with_limit():
     tm.assert_series_equal(result, expected)
 
 
-def test_nearest_upsample_with_limit():
-    rng = date_range("1/1/2000", periods=3, freq="5t")
+@pytest.mark.parametrize("freq", ["Y", "10M", "5D", "10H", "5Min", "10S"])
```

Comment:
you need to add the tz_aware_fixture here as well

Reply:
Thanks @jreback. I've done.